CONTRIBUTING.md

```diff
@@ -0,0 +1,61 @@
+# Contributing to bible memorization
+
+- [Contributing to bible memorization](#contributing-to-bible-memorization)
+  - [Getting Started](#getting-started)
+  - [Pulls Requests](#pulls-requests)
+  - [Submitting a Pull Request](#submitting-a-pull-request)
+
+## Getting Started
+
+Start by **starring** the repo or [forking](https://github.com/garretlin/bible-memorization/fork) it.
```

Comment:
I thought you start by cloning the rep, not just starring/forking it... But I don't know if people will be working directly through the Github editor lol

Reply:
will fix, this was from a template -- and we would not want people to be working directly through Github editor lol

Reply:
actually, I remember what this was about -- a little confusing. For "getting started" it's not necessarily for only for contributing, but if someone were to take our repo and try it out.

then if they are to contribute by making code changes and making PR, they would follow the steps in `Submitting a Pull Request`